docs/polkadot/implementors-guide/guide.md

```diff
@@ -0,0 +1,854 @@
+                    /                                  \
+                    |            Runtime               |
+                    |                                  |
+                    \_________(Runtime API )___________/
+                                |       ^
+                                V       |
+               +----------------------------------------------+
+               |                                              |
+               |                   Node                       |
+               |                                              |
+               |                                              |
+               +----------------------------------------------+
+                                   +  +
+                                   |  |
+               --------------------+  +------------------------
+                                 Transport
+               ------------------------------------------------
+
+```
+
+
+It is also helpful to divide Node-side behavior into two further categories: Networking and Core. Networking behaviors relate to how information is distributed between nodes. Core behaviors relate to internal work that a specific node does. These two categories of behavior often interact, but can be heavily abstracted from each other. Core behaviors care that information is distributed and received, but not the internal details of how distribution and receipt function. Networking behaviors act on requests for distribution or fetching of information, but are not concerned with how the information is used afterwards. This allows us to create clean boundaries between Core and Networking activities, improving the modularity of the code.
+
+```
+          ___________________                    ____________________
+         /       Core        \                  /     Networking     \
+         |                   |  Send "Hello"    |                    |
+         |                   |-  to "foo"   --->|                    |
+         |                   |                  |                    |
+         |                   |                  |                    |
+         |                   |                  |                    |
+         |                   |    Got "World"   |                    |
+         |                   |<--  from "bar" --|                    |
+         |                   |                  |                    |
+         \___________________/                  \____________________/
+                                                   ______| |______
+                                                   ___Transport___
+
+```                                                   
+
+
+Node-side behavior is split up into various Processes. Processes are long-lived workers that perform a particular category of work. Processes can communicate with each other, and typically do so via an Overseer that prevents race conditions.
```

Comment:
Yeah, this wording is strange to me as well. Perhaps calling them something like 'Systems' or 'Subsystems' would work better?

Reply:
Let's rename them then because I've had the same feeling that there's too much potential confusion between Process (parachain host) and Process (OS). 
 
I believe @montekki was advocating for "Worker" in the past. I'm not strongly opinionated on either that or Subsystem